Mobile Client/PrimeFileSync/app/src/main/java/com/example/primefilesync/EditFragment.java

```diff
@@ -76,11 +85,34 @@ public void onClick(View v) {
             }
         });
 
+        // set up Download button
+
+        Button downloadbtn = (Button) view.findViewById(R.id.Downloadbutton);
+            downloadbtn.setOnClickListener(new View.OnClickListener() {
+                @RequiresApi(api = Build.VERSION_CODES.JELLY_BEAN)
+                @Override
+                public void onClick(View view) {
+                    downloadManager = (DownloadManager)getActivity().getSystemService(Context.DOWNLOAD_SERVICE);
+                    Uri uri = Uri.parse("http://10.40.6.85:3003/files/" + fileName.getText());
```

Comment:
Replace the IP address with the URL of the server within the Heroku cloud; this line of code becomes 
```suggestion
                    Uri uri = Uri.parse("https://rocky-plateau-19773.herokuapp.com/download/" + fileName.getText());
```